packages/salesforcedx-vscode-soql/package.json

```diff
@@ -121,6 +121,20 @@
         "scopeName": "source.soql",
         "path": "./node_modules/@salesforce/apex-tmlanguage/grammars/soql.tmLanguage"
       }
-    ]
+    ],
+    "commands": [
+      {
+        "command": "soql.builder.open.new",
+        "title": "SOQL: New Query"
```

Comment:
Should the command title include the word "File" ? 🤔

Reply:
Throwing this out as a discussion point. Should the emphasis be on creating the file or the fact that we're launching SOQL Builder. This will be the new entry point, yes? So should it be SOQL: Open Builder or SOQL: Open SOQL Builder (I don't know if any of our commands use Launch or if that's an ok option based on our UI standards.

Reply:
When I open command palette and do a search, "Open" seems to be the most popular choice.

Reply:
I'm ready to defer to @ekapner on this one.

Reply:
@ekapner This functionality is one of two(or more) ways to get to the builder but not necessarily a primary entry point. The intention is to create an easy pathway for customers to use the SOQL Builder if they don't have an existing `.soql` file.
So there is a distinction between
1. Opening and _existing_ file with the builder(current functionality) `SOQL: Open file with SOQL Builder` `SOQL: Open current editor with SOQL Builder`
2. Creating a new .soql file and using the builder create a query (this PR) `SOQL: Start a new query with SOQL Builder` `SOQL: Create a new Query with SOQL Builder`
3. Just a single command to "open" the builder (does not make sense) ' SOQL: Open Builder'

Because the builder is really an editor it always needs to be open with the context of a file, whether its new or existing.

Reply:
Thanks, @jonnyhork, for the thorough explanation. Makes sense. 

So...how many menu items do we need? Throwing these out:
SOQL: Open File in SOQL Builder
SOQL: Open File in Text Editor
SOQL: Create a Query in SOQL Builder (unsure the distinction re: Start a new query vs Create a new...)

Reply:
Sorry for the confusion, the commands in my comment were just examples of what wording might look like for each option. Not to be taken as unique commands that we will have.
For this PR we only need one for option 2, whatever wording you want =)

Reply:
What do you all think about these? I'm torn. But I like clarifying that we're talking soql builder. Looking at other VS code commands, let's get rid of the articles. Leaning toward Create.
SOQL: Create Query in SOQL Builder
SOQL: New Query in SOQL Builder

Reply:
I like 'Create'